registry.k8s.io/images/k8s-staging-cluster-api-azure/images.yaml

```diff
@@ -94,6 +94,7 @@
     "sha256:95a2400fb3aeeeae994829c094ece2a9933e4e2221673753a999947c1728b4af": ["v1.19.1"]
     "sha256:64a2c3c2483f30004b959d57152b722d00d21072db31494c41a2ffe37a3bac58": ["v1.19.2"]
     "sha256:9a901756982bbe52c377d40d825c5eb5233a059584ce6bf1c13d308182fd9f8a": ["v1.19.3"]
+    "sha256:434b01edb61b2b40ffc05a3c8ef14854475e16f81cfecf26bd43e76183da2573": ["v1.19.4"]
```

Comment:
SHA Matches with one present in staging repository. https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-azure/global/cluster-api-azure-controller@sha256:434b01edb61b2b40ffc05a3c8ef14854475e16f81cfecf26bd43e76183da2573/details?inv=1&invt=AbxNmw